styles.css

```diff
@@ -0,0 +1,97 @@
+.container {
+    max-width: 1200px;
+}
+.row {
+    clear: both;
+}
+.column {
+    width: 25%;
+    float: left;
+}
+body {
+    margin: none;
+}
+.col-10 {
+    width: 100%
+}
+.navbar {
+    width: 100%;
+    
+    background-color: black;
+    font-family: Arial;
+}
+.navbar a {
+    float: left;
+    color: white;
```

Comment:
You may want to make the navbar black, so this shows in the browser correctly.
Currently, this whole navbar appears as a blank white space